kubetest/e2e.go

```diff
@@ -640,6 +651,11 @@ func kubemarkUp(dump string, o options, deploy deployer) error {
 		return err
 	}
 
+	// Remember main cluster kubeconfig, this nescessary for dumping logs with logexporter.
+	if err := os.Setenv("KUBECONFIG_MAIN", os.Getenv("KUBECONFIG")); err != nil {
```

Comment:
Nit: Some other places using 'KUBEMARK_ROOT_KUBECONFIG' as a env variable decscribing "ROOT" cluster's kubeconfig in kubemark. I think we may want to be consitent with them.